corpus/accounts/models.py

```diff
@@ -151,3 +155,22 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return f"{self.user.first_name} {self.user.last_name} [{self.sig.name}]"
+
+
+class Core(models.Model):
+    user = models.OneToOneField(User, null=False, on_delete=models.CASCADE)
+    post = models.CharField(max_length=100, null=False)
+    sig = models.ForeignKey(SIG, null=False, on_delete=models.CASCADE)
+    term_start = models.DateField(default=now, null=False)
+    term_end = models.DateField(null=True)
```

Comment:
I may have specified this `null=true` constraint when we planned on having Faculty in the same model, but I think we can remove this.
Every student core member will NEED to have a start and end date to their term.